apps/src/templates/PaneHeader.jsx

```diff
@@ -131,6 +131,7 @@ var PaneButton = function (props) {
     <div
       id={props.id}
       style={divStyle}
+      onClick={props.onClick}
```

Comment:
How is `onClick` a new property here?  Did we never have a `PaneButton` that actually did something before?